modules/localization/msf/msf_localization.cc

```diff
@@ -306,6 +306,9 @@ void MSFLocalization::OnGnssHeading(
 
 void MSFLocalization::OnGps() {
   std::unique_lock<std::mutex> lock(mutex_imu_msg_);
+  if (!raw_imu_msg_) {
+    return;
+  }
```

Comment:
A better practice is to check the message before the lock.

Reply:
OK

Reply:
> A better practice is to check the message before the lock.

OK